TRex2-lib/src/Packets/RulePkt.h

```diff
@@ -320,17 +320,17 @@ class RulePkt {
   // Identifier of the rule
   int ruleId;
   // Array of event predicates
-  std::map<int, Predicate> predicates;
+  std::vector<Predicate> predicates;
```

Comment:
Change `maps` to `vectors`, because (as far I was able to tell) there was no need of them and the resulting code is simpler and probably more efficient.